SPECS/kubevirt/kubevirt.spec

```diff
@@ -216,6 +217,9 @@ install -p -m 0644 cmd/virt-handler/nsswitch.conf %{buildroot}%{_datadir}/kube-v
 %{_bindir}/virt-tests
 
 %changelog
+* Fri May 17 2024 Rohit Rawat <rohitrawat@microsoft.com> - 0.59.0-18
```

Comment:
This change was added in release 17 in fast-track - please move it there. You can modify/re-order changes, which are only present in `main`, since they haven't been published, yet.